sim_pipeline/gg_lens.py

```diff
@@ -208,7 +208,12 @@ def los_linear_distortions(self):
         # TODO: more realistic distribution of shear and convergence,
         #  the covariances among them and redshift correlations
         if not hasattr(self, '_gamma'):
-            gamma = np.random.normal(loc=0, scale=0.1)
+            mixture = GaussianMixtureModel(
+                means=[0.00330796, -0.07635054, 0.11829008],
```

Comment:
there are many hard-coded values here. There is no information about from where these values are coming from. Perhaps put them in a configuration file (or class) with a description of what they are.
Also, the user (which might be you or someone else) might want to have different options. So you could define an input option (with a default setting).